bedrock/externalpages/templates/externalpages/pocket/includes/mobile-nav.html

```diff
@@ -0,0 +1,79 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="mobile-nav-wrapper" tabindex="-1">
+    <nav class="mobile-nav" role="dialog" aria-modal="true" aria-label="Pocket Mobile Menu">
+        <div class="mobile-nav-close">
+            <button class="mobile-nav-close-btn" aria-label="Close the Pocket Mobile Menu" data-tooltip="Close">
+                <svg aria-labelledby="mobile-nav-close-label" fill="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M14.707 4.293a1 1 0 0 1 0 1.414L8.414 12l6.293 6.293a1 1 0 0 1-1.414 1.414l-7-7a1 1 0 0 1 0-1.414l7-7a1 1 0 0 1 1.414 0Z"></path></svg>
+                <span id="mobile-nav-close-label">Close the Pocket Mobile Menu</span>
+            </button>
```

Comment:
Seeing as these buttons already contain good readable copy, I don't think the `aria-label` and `aria-labelledby` attributes are needed?